### PR TITLE
Ratings func

### PR DESCRIPTION
**Pull request by CHaiz15**
## Type of Change

  - Added delete and post of the user's rating. The individual post and delete functions work but the functionality does not.

## Description

  - Added delete and post functions for the users ratings on apiCalls.

## Items for Review	

  - apiCalls.js. movieDetails.js

### CODE REVIEW

File: src/MovieDetails/MovieDetails.js [hunk 1]
```diff
@@ -1,25 +1,43 @@
...
+      currentMovieRating: 0,
+    }
+  }
+
+  handleChange = (e) => {
+    e.preventDefault()
+    this.setState({currentMovieRating: parseInt(e.target.value)})
+  }
+
+  updateRatings = () => {
```
Comment (CHaiz15) [inline src/MovieDetails/MovieDetails.js:20]: This is broken :(

File: src/MovieDetails/MovieDetails.scss [hunk 2]
```diff
@@ -13,7 +13,7 @@
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
-  height: 50rem;
+  height: 41.2rem;
```
Comment (kimmccaskill) [inline src/MovieDetails/MovieDetails.scss:16]: So specific. Interesting

File: src/MovieThumbnail/MovieThumbnail.js [hunk 3]
```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import './MovieThumbnail.scss'
 
 const MovieThumbnail = ({ title, average_rating, backdrop_path, ratings }) => {
+
   return (
-    <span>
+    <span onClick={() => window.scrollTo(0, 0)}>
```
Comment (kimmccaskill) [inline src/MovieThumbnail/MovieThumbnail.js:7]: Clever!

File: src/MovieDetails/MovieDetails.js (hunk 1 above)
Comment (kimmccaskill) [reply src/MovieDetails/MovieDetails.js:20]: We'll take a gander

**Pull request merged**